README.md

```diff
@@ -93,7 +93,11 @@ Get whether the color is "light" or "dark", useful for deciding text color.
 color.negate()         // rgb(0, 100, 255) -> rgb(255, 155, 0)
 
 color.lighten(0.5)     // hsl(100, 50%, 50%) -> hsl(100, 50%, 75%)
+color.lighten(0.1)     // hsl(100, 50%, 10%) -> hsl(100, 50%, 11%)
 color.darken(0.5)      // hsl(100, 50%, 50%) -> hsl(100, 50%, 25%)
+color.darken(0.1)      // hsl(100, 50%, 10%) -> hsl(100, 50%, 9%)
```

Comment:
I don't see the point of these two additions; the 50% ratio should already be clear. I'd argue the 10% ratio is a bit confusing.

Reply:
With additional examples methods are more clearly because people read fast and can miss something. I think this is one of reasons why #166 comes. The second reason is people don't know about color models enough. They think that `darken()` and `whiten()` methods make a color darken or whiten in each of cases. But this is not. Additionally docs withdescription about color models might be nice.

Reply:
May be are there should be something like this:
```javascript
color.lighten(0.5)     // hsl(100, 50%, 50%) -> hsl(100, 50%, 75%)
color.lighten(0.5)     // hsl(100, 50%, 0)      -> hsl(100, 50%, 0)
color.darken(0.5)      // hsl(100, 50%, 50%) -> hsl(100, 50%, 25%)
color.darken(0.5)      // hsl(100, 50%, 0)      -> hsl(100, 50%, 0)
```

Reply:
That would be fine, as long as the alignment is fixed ^^

Reply:
> That would be fine, as long as the alignment is fixed ^^

Done with new commit